### ISSUE TITLE
Yubico Authenticator cannot be installed from the Microsoft Store on most versions of Windows 10

### ISSUE THREAD

**Issue by gurnec** [Yubico/yubioath-desktop#613]
- Yubico Authenticator version: Microsoft Store version released Mar/5/2020
- Operating system and version: Windows 10 1909 (Build 10.0.18363.836)

This isn't exactly a software bug, but rather an issue with how Yubico Authenticator was submitted to the Microsoft Store. Below are the requirements currently displayed in the Store for this app on my PC:

![image](https://user-images.githubusercontent.com/2966862/83356408-fe1a0880-a333-11ea-9396-db1b65e226b2.png)

It looks like the Store expects one to submit the minimum Windows *Build number* for the app, however this app was submitted with a minimum of `19030.0` which, as a *Build number*, doesn't exist*. I suspect the intent was to submit a minimum *Version* of 1903, the version of Windows released May 2019, which has the [*Build number* 18362](https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1903_(May_2019_Update)). If that's true, then the Store should be updated with this minimum Build number.

(* Technically that build number 19030 does exist, but it was a Preview Build of version 2004 released only on the fast ring; I doubt that's the minimum that was intended to be set in the Store though.)

**Comment by dagheyman** [comment]
Hi, thanks for the report. The minimun required build version is correct, since there seems to be a bug (link to feedback hub https://aka.ms/AA7uqo2 ) in earlier versions of Windows related to how store apps handle registry settings.

**Comment by gurnec** [comment]
> The minimun required build version is correct

My apologies if I sound argumentative, but I believe one of us is misunderstanding something.

The minimum Build number of Yubico Authenticator is listed on the store as `19030`. For reference, here are all of the (non-Preview) versions of Windows 10, according to Wikipedia:

| Version | Codename    | Marketing name       | **Build** | Release date      | 
|:-------:|-------------|----------------------|:---------:|------------------:| 
| 1507    | Threshold 1 | N/A                  | **10240** | July 29, 2015     | 
| 1511    | Threshold 2 | November Update      | **10586** | November 10, 2015 | 
| 1607    | Redstone 1  | Anniversary Update   | **14393** | August 2, 2016    | 
| 1703    | Redstone 2  | Creators Update      | **15063** | April 5, 2017     | 
| 1709    | Redstone 3  | Fall Creators Update | **16299** | October 17, 2017  | 
| 1803    | Redstone 4  | April 2018 Update    | **17134** | April 30, 2018    | 
| 1809    | Redstone 5  | October 2018 Update  | **17763** | November 13, 2018 | 
| 1903    | 19H1        | May 2019 Update      | **18362** | May 21, 2019      | 
| 1909    | 19H2        | November 2019 Update | **18363** | November 12, 2019 | 
| 2004    | 20H1        | May 2020 Update      | **19041** | May 27, 2020      | 

Are you saying that Yubico Authenticator, which was released on the Microsoft Store on March 5th, requires a build of Windows, `19030`, which wasn't available for general release until 6 days ago? This really doesn't seem correct to me...

**Comment by dagheyman** [comment]
> Are you saying that Yubico Authenticator, which was released on the Microsoft Store on March 5th, requires a build of Windows, 19030, which wasn't available for general release until 6 days ago?

Yes.

While packaging for the Microsoft Store, we discovered that the package did not work correctly on the current Windows 10 version, but worked fine in the insider builds. We tracked it down to an issue in Windows related to how registry settings are handled in store apps, as mentioned above. Rather than waiting for 20.04 to be generally available we decided to publish the package with the minimum build version set, to make it available for insiders for testing. This seem to have caused some confusion, and could probably have been communicated better.

To summarise, the version in Microsoft Store requires Windows 10 20.04, all older versions of Windows should use the .msi package.

**Comment by dagheyman** [comment]
For the next release we should probably set the minimum build number to 19041 though to make it clear we require Windows 10 20.04.

**Comment by gurnec** [comment]
Thank you for clarifying my misunderstanding! My apologies for using up your time.
